src/components/Form/Form.component.tsx

```diff
@@ -0,0 +1,14 @@
+import './Form.style.scss';
+
+const Form = ({ children }: any) => {
```

Comment:
Ici j'aurais rajouté un props onSubmit pour le trigger apres le preventDefault.

Reply:
Yep !

Reply:
Quand tu submit ton form tu trigger une action, par exemple l'envoi de la requete pour login.
Ici tu as un form qui ne fait rien on submit, et l'envoi de la requête tu l'as mise sur le onClick du bouton dans ton Signup.component.
onClick et onSubmit sont deux event differents, et là en l'occurence pour envoyer ta requête tu soumet le formulaire, donc c'est mieux de le faire ici, in fine le résultat est le même, mais sémantiquement c'est plus juste.

```suggestion
const Form = ({ children, onSubmitForm }: any) => {
     function onSubmit (e: any) {
        e.preventDefault();
        onSubmitForm(); // cette méthode sera chargée d'envoyer la requête
    }
}
```